packages/manager/src/components/TabbedPanel/TabbedPanel.tsx

```diff
@@ -89,8 +92,14 @@ const TabbedPanel = React.memo((props: TabbedPanelProps) => {
         <StyledTabs index={tabIndex} onChange={tabChangeHandler}>
           <StyledTabList>
             {tabs.map((tab, idx) => (
-              <StyledTab key={`tabs-${tab.title}-${idx}`}>
+              <StyledTab
+                disabled={tab.disabled}
+                key={`tabs-${tab.title}-${idx}`}
+              >
                 {tab.title}
+                {tab.disabled && props.tabDisabledMessage && (
+                  <TooltipIcon status="help" text={props.tabDisabledMessage} />
```

Comment:
![Screenshot 2024-01-05 at 1 41 34 PM](https://github.com/linode/manager/assets/115251059/6bfa72a7-3a5d-46a5-a799-b6e053ad0f11)


This component is caused the height of our Tabs to be extra tall. This may fix that: 

```suggestion
                  <TooltipIcon sxTooltipIcon={{ p: 0, ml: 1 }} status="help" text={props.tabDisabledMessage} />
```